stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.16.0
+version: 1.16.1
```

Comment:
As this is a new feature lets bump the minor version here. `1.17.0`
After that bump this change looks good!

Reply:
Bumped that now - thanks!